changelog/depercate_object_selector.dd

```diff
@@ -0,0 +1,26 @@
+Deprecated `object.selector`
+
+The public import of `selector` in the module `object` has been deprecated.
+Please explicitly import `selector` instead using: `import core.attribute : selector;`
+
+The following code has been deprecated:
+
+```
```

Comment:
Shouldn't we use `---` with ddoc?

Reply:
Possibly. Is markdown enabled for changelog entires.

Reply:
[Yes](https://github.com/dlang/druntime/blob/40d5de0c52a85f1458c97e119bcde2846c6dd7f5/posix.mak#L68).

Reply:
Then this is fine.

Reply:
Yup - http://dtest.dlang.io/artifact/website-403dc89ebaba38a7ee42cfbb6c2b6c5f8657bffd-e9d3a6d3493d65a0e0bec935c528c07a/web/changelog/pending.html#depercate_object_selector

For future reference: you should even be able to use .md as an extension.